service/src/main/java/org/apache/griffin/core/metastore/hive/HiveMetaStoreController.java

```diff
@@ -34,6 +35,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 public class HiveMetaStoreController {
 
     @Autowired
+    @Qualifier(value = "hive_metastore")
```

Comment:
Pls use lowerCamelCase to name variables. The following code is also the same.

Reply:
done